docs/installing-solidity.rst

```diff
@@ -6,6 +6,28 @@
 Installing Solidity
 ###################
 
+Versioning
```

Comment:
How about this split? Probably the second part on the bottom should go into the dev (and not installation) section?